pkg/state/conformance/state.go

```diff
@@ -292,15 +285,20 @@ func (suite *StateSuite) TestWatchKind() {
 	case event := <-chWithBootstrap:
 		suite.Assert().Equal(state.Updated, event.Type)
 		suite.Assert().Equal(resource.String(path2), resource.String(event.Resource))
-		suite.Assert().Equal(path2.Metadata().Version(), event.Resource.Metadata().Version())
+		suite.Assert().Equal(resource.NewVersion(3), event.Resource.Metadata().Version())
 	case <-time.After(time.Second):
 		suite.FailNow("timed out waiting for event")
 	}
 
 	// get event history
 	chWithTail := make(chan state.Event)
 
-	suite.Require().NoError(suite.State.WatchKind(ctx, path1.Metadata(), chWithTail, state.WithKindTailEvents(1000)))
+	err = suite.State.WatchKind(ctx, path1.Metadata(), chWithTail, state.WithKindTailEvents(1000))
```

Comment:
probably it makes more sense to split the test and `t.Skip()` it if not supported? it's more explicit and visible?

Reply:
done